src/core/qgsmimedatautils.cpp

```diff
@@ -206,11 +206,13 @@ QByteArray QgsMimeDataUtils::layerTreeNodesToUriList( const QList<QgsLayerTreeNo
 QString QgsMimeDataUtils::encode( const QStringList &items )
 {
   QString encoded;
+  // Do not escape colon twice
+  QRegExp re( "([^\\\\]):" );
```

Comment:
Use the recommended replacement QRegularExpression instead.

This could possibly be made a bit cleaner by a negative look behind.